src/chiron_utils/bots/__init__.py

```diff
@@ -49,8 +49,10 @@
     )
 if {"torch", "transformers"} < importable_modules:
     from chiron_utils.bots.llm_advisor_bot import LlmAdvisor as LlmAdvisor
+    from chiron_utils.bots.llm_advisor_new_bot import LlmNewAdvisor as LlmNewAdvisor
 
     BOTS.extend([LlmAdvisor])
+    BOTS.extend([LlmNewAdvisor])
```

Comment:
Let's merge these lines:

```suggestion
    BOTS.extend(
        [
            LlmAdvisor,
            LlmNewAdvisor,
        ]
    )
```